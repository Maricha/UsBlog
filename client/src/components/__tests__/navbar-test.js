import React from 'react';
import { shallow } from 'enzyme'; 
import Navbar from '../navbar';

describe('navbar Component', () => {
  it('renders correctly', () => {
    const component = shallow(<Navbar
        classes={{sd: "sd"}}    
        title="title"
        toolbar
        mainUrl="/"
        contact
        adminLogout
    />);
    expect(component).toMatchSnapshot();
  })
});