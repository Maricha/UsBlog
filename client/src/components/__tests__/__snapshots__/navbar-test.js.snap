// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`navbar Component renders correctly 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <WithStyles(Component)
    adminLogout={true}
    classes={
      Object {
        "sd": "sd",
      }
    }
    contact={true}
    mainUrl="/"
    title="title"
    toolbar={true}
  />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "function",
    "props": Object {
      "adminLogout": true,
      "classes": Object {
        "link": "Component-link-5",
        "sd": "undefined sd",
        "titleLink": "Component-titleLink-4",
        "toolbarMain": "Component-toolbarMain-1",
        "toolbarSecondary": "Component-toolbarSecondary-3",
        "toolbarTitle": "Component-toolbarTitle-2",
      },
      "contact": true,
      "mainUrl": "/",
      "title": "title",
      "toolbar": true,
    },
    "ref": null,
    "rendered": null,
    "type": Object {
      "$$typeof": Symbol(react.memo),
      "compare": null,
      "propTypes": Object {
        "classes": [Function],
        "title": [Function],
      },
      "type": [Function],
    },
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "function",
      "props": Object {
        "adminLogout": true,
        "classes": Object {
          "link": "Component-link-5",
          "sd": "undefined sd",
          "titleLink": "Component-titleLink-4",
          "toolbarMain": "Component-toolbarMain-1",
          "toolbarSecondary": "Component-toolbarSecondary-3",
          "toolbarTitle": "Component-toolbarTitle-2",
        },
        "contact": true,
        "mainUrl": "/",
        "title": "title",
        "toolbar": true,
      },
      "ref": null,
      "rendered": null,
      "type": Object {
        "$$typeof": Symbol(react.memo),
        "compare": null,
        "propTypes": Object {
          "classes": [Function],
          "title": [Function],
        },
        "type": [Function],
      },
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
